src/test/java/teammates/test/cases/action/InstructorFeedbackEditCopyActionTest.java

```diff
@@ -268,16 +268,16 @@ public void testExecuteAndPostProcess() {
 
         assertEquals("", editCopyData.redirectUrl);
 
-        expectedString = "\"\" is not acceptable to TEAMMATES as a/an feedback session name because it is empty. "
+        expectedString = "The field feedback session name is empty. "
```

Comment:
This issue might apply to, not just cases where the field is empty. Does it? If it is, then it goes beyond the scope of this PR?.